exonum-java-binding/testkit/src/main/java/com/exonum/binding/testkit/FakeTimeProvider.java

```diff
@@ -16,25 +16,59 @@
 
 package com.exonum.binding.testkit;
 
+import static com.google.common.base.Preconditions.checkArgument;
+
+import java.time.ZoneOffset;
 import java.time.ZonedDateTime;
+import java.time.temporal.TemporalAmount;
 
-// TODO: update Javadocs in P2 [ECR-3051]
+/**
+ * Fake time provider for service testing. Allows to manually manipulate time that is returned
+ * by TestKit time service.
+ */
```

Comment:
\+ If you need to set results of different consecutive calls on (link) getTime,
 consider using a mock of TimeProvider instead.